GameServerLib/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -160,7 +160,7 @@ public bool CanMove()
         {
             return !HasCrowdControl(CrowdControlType.STUN) &&
                 !IsDashing &&
-                !IsCastingSpell &&
+                (GetBuffs().Exists(x => x.OriginSpell.SpellData.CanMoveWhileChanneling) || !IsCastingSpell) &&
```

Comment:
This still looks incorrect to me (as if you cast a spell that allows you to move and one that don't it will be very not so correct).